templates/iiif-images-compose.ecs.yml

```diff
@@ -9,3 +9,8 @@ services:
         awslogs-stream-prefix: iiif_image
     environment:
       HONEYBADGER_API_KEY_IMAGESERVER:
+      AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
+      AWS_SECRET_KEY: ${AWS_SECRET_ACCESS_KEY}
+      S3CACHE_BUCKET_NAME: ${S3_SOURCE_BUCKET_NAME}/cantaloupe/cache #NO TRAILING SLASH!
```

Comment:
isn't this inherited from the base template?

Reply:
It is and this was updated by Keith in a later version of this file.